Sources/SwiftTranslate/TranslationServices/OpenAITranslator.swift

```diff
@@ -13,42 +13,52 @@ struct OpenAITranslator {
     // MARK: Private
     
     private let openAI: OpenAI
-    private let model: OpenAIModel
-    
+    private let model: Model
+
     // MARK: Lifecycle
     
-    init(with apiToken: String, model: OpenAIModel = .gpt3_5TurboInstruct) {
-        self.openAI = OpenAI(apiToken: apiToken)
+    init(with apiToken: String, model: Model) {
+        let configuration = OpenAI.Configuration(
+            token: apiToken,
+            timeoutInterval: 60
+        )
+        self.openAI = OpenAI(configuration: configuration)
         self.model = model
     }
     
     // MARK: Helpers
-    
-    private func completionQuery(for translatableText: String, targetLanguage: Language, comment: String?) -> CompletionsQuery {
-        var prompt =
+
+    private func chatQuery(for inputText: String, targetLanguage: Language, comment: String?) -> ChatQuery {
+        let systemPrompt = """
+            You are a helpful assistant designed to output JSON.
+            Your task is to translate text.
+            If the input text contains argument placeholders (%arg, @arg1, %lld, etc), it's important they are preserved in the translated text.
             """
-            Translate the text between the backticks (``````) from English to the language with ISO code: \(targetLanguage.rawValue)
-            - DO NOT translate the prompt or any other text that is not inside the backticks (``````).
-            - DO NOT INCLUDE backticks (``````) in your response!
-            - DO NOT REMOVE argument placeholders from your response! (%arg, @arg1, %lld, etc)
+
+        var userPrompt =
+            """
+            Translate the text between the backticks (``````) from English to the language with ISO 639-1 code: \(targetLanguage.rawValue).
```

Comment:
Just a heads up. This prompt change might have made the performance worse when using gpt-3.5, not sure, but I thought it performed rather poorly when I later tried it out on some of my actual string catalogs. I thought it 
did a good job (in general) using gpt-4o though. 

But gpt-4o still made a lot of mistakes with whitespace and "special" strings for me. Maybe I should stop constructing strings like `Text("Prices are ") + Text("per kWh").bold()`, because it seemed it often struggled to get those leading/trailing whitespaces correct. (Just recently found out markdown is supported now, so I should try and refactor to that to get rid of those "sub strings" which understandably should be much harder for gpt to get right).

Anyhow, I think it would be a good idea to add some tests, similar to what I added in #31, but for the translator. Just to have a easy way to evaluate translation performance when making changes or when trying to handle some incorrect behavior. 

I started adding this last weekend, as a continuation of #31. I'll try and tidy it up and put up an PR for it soon.

Reply:
You should use `Text("Prices are \(someValue) per KwH")`. This automatically generates a string with the `%_` placeholder and ensures translations work correctly.